sample/Kmd.Logic.Cpr.Client.Sample/Program.cs

```diff
@@ -96,6 +102,45 @@ private static async Task Run(AppConfiguration configuration)
                 var citizen = await cprClient.GetCitizenByCprAsync(configuration.CprNumber).ConfigureAwait(false);
 
                 Log.Information("Citizen data: {@Citizen}", citizen);
+
+                var isSubscribeByCprSuccess = await cprClient.SubscribeByCprAsync(configuration.CprNumber).ConfigureAwait(false);
+
+                if (!isSubscribeByCprSuccess)
+                {
+                    Log.Error("Invalid CPR Number {@CprNumber}", configuration.CprNumber);
+                }
+
+                Log.Information("Subscribed successfully for CprNumber {CprNumber}", configuration.CprNumber);
+
+                var isSubscribeByIdSuccess = await cprClient.SubscribeByIdAsync(cprPersonId).ConfigureAwait(false);
+
+                if (!isSubscribeByIdSuccess)
+                {
+                    Log.Error("Invalid CPR PersonId {personId}", cprPersonId);
+                }
+
+                Log.Information("Subscribed successfully for personId {personId}", cprPersonId);
+
+                var isUnsubscribeByCprSuccessful = await cprClient.UnsubscribeByCprAsync(configuration.CprNumber).ConfigureAwait(false);
+
+                if (isUnsubscribeByCprSuccessful)
+                {
+                    Log.Information("Unsubscribed successfully for CprNumber {CprNumber}", configuration.CprNumber);
+                }
+
+                var isUnsubscribeByIdSuccessful = await cprClient.UnsubscribeByIdAsync(cprPersonId).ConfigureAwait(false);
+
+                if (isUnsubscribeByIdSuccessful)
+                {
+                    Log.Information("Unsubscribed successfully for personId {personId}", cprPersonId);
+                }
+
+                var citizenList = await cprClient.GetAllCprEvents(datefrom, dateTo, pageNo, pageSize).ConfigureAwait(false);
```

Comment:
This is the unfiltered query. I welcome you calling it but I would put it before all the subscribe/unsubscribe logic

Reply:
Added this method at the beginning .